src/NzbDrone.Core/Download/Clients/Tribler/Models/TriblerDownloadClientApi.cs

```diff
@@ -0,0 +1,226 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NzbDrone.Core.Download.Clients.Tribler
+{
+    public enum DownloadStatus
+    {
+        [EnumMember(Value = @"WAITING4HASHCHECK")]
+        Waiting4HashCheck = 0,
+
+        [EnumMember(Value = @"HASHCHECKING")]
+        Hashchecking = 1,
+
+        [EnumMember(Value = @"METADATA")]
+        Metadata = 2,
+
+        [EnumMember(Value = @"DOWNLOADING")]
+        Downloading = 3,
+
+        [EnumMember(Value = @"SEEDING")]
+        Seeding = 4,
+
+        [EnumMember(Value = @"STOPPED")]
+        Stopped = 5,
+
+        [EnumMember(Value = @"ALLOCATING_DISKSPACE")]
+        AllocatingDiskspace = 6,
+
+        [EnumMember(Value = @"EXIT_NODES")]
+        Exitnodes = 7,
+
+        [EnumMember(Value = @"CIRCUITS")]
+        Circuits = 8,
+
+        [EnumMember(Value = @"STOPPED_ON_ERROR")]
+        StoppedOnError = 9,
+    }
+
+    public class Trackers
+    {
+        [JsonProperty("url", NullValueHandling = NullValueHandling.Ignore)]
+        public string Url { get; set; }
+
+        [JsonProperty("peers", NullValueHandling = NullValueHandling.Ignore)]
+        public object Peers { get; set; }
+
+        [JsonProperty("status", NullValueHandling = NullValueHandling.Ignore)]
+        public string Status { get; set; }
+    }
+
+    public class Download
+    {
+        [JsonProperty("name", NullValueHandling = NullValueHandling.Ignore)]
+        public string Name { get; set; }
+
+        [JsonProperty("progress", NullValueHandling = NullValueHandling.Ignore)]
+        public float? Progress { get; set; }
+
+        [JsonProperty("infohash", NullValueHandling = NullValueHandling.Ignore)]
+        public string Infohash { get; set; }
+
+        [JsonProperty("anon_download", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
This is an example of a property that does need the name defined, but shouldn't need `NullValueHandling`